stencil.config.ts

```diff
@@ -22,6 +22,14 @@ export const config: Config = {
           src: './components/test/*.svg',
           dest: './assets/',
         },
+        {
```

Comment:
This change was to allow the local dev server to have more than the single `index.html` test template. Is there another way to accomplish this or is this the recommended path?